controllers/apartments_controller.rb

```diff
@@ -0,0 +1,44 @@
+get "/apartments" do
+  apartments = Apartment.all
+  @apartments = apartments
+  erb :"/apartments/list_apartments"
+end
+
+get "/apartments/new" do
+  apartments = Apartment.all
+  @apartments = apartments
+  erb :"apartments/add_apartment"
+end
+
+post "/apartments/new" do
+  apartments = Apartment.all
+  @address = params[:address]
+  @monthly_rent = params[:monthly_rent]
+  @sqft = params[:sqft]
+  @num_beds = params[:num_beds]
+  @num_baths = params[:num_baths]
+  new_apartment = Apartment.create(address: @address, monthly_rent: @monthly_rent, sqft: @sqft, num_beds: @num_beds, num_baths: @num_baths)
```

Comment:
None of these really need to be instance variables since you're not referencing them in a view. In fact, they don't really even need to be variables! I'd just do `Apartment.create(address: params[:address]...)`. It'd be a lot less to type!